src/group_theory/nilpotent.lean

```diff
@@ -321,3 +351,19 @@ begin
     rcases (h y') with ⟨y, rfl⟩,
     simpa using hd (mem_map_of_mem f (hx y)) }
 end
+
+lemma lower_central_series.map {H : Type*} [group H] (f : G →* H) (n : ℕ) :
+  subgroup.map f (lower_central_series G n) ≤ lower_central_series H n :=
+begin
+  induction n with d hd,
+  { simp [nat.nat_zero_eq_zero] },
+  { rintros a ⟨x, hx : x ∈ lower_central_series G d.succ, rfl⟩,
+    refine closure_induction hx _ (by simp [f.map_one, subgroup.one_mem _])
+      (λ y z hy hz, by simp [monoid_hom.map_mul, subgroup.mul_mem _ hy hz])
+      (λ y hy, by simp [f.map_inv, subgroup.inv_mem _ hy]),
```

Comment:
You still only have one goal after this tactic so you don't need the { on the next line.